File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/unions/unionTypesWithoutInterfaceCanDeserialize/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.unions.unionTypesWithoutInterfaceCanDeserialize.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Search = "search";

    public static class SEARCH_INPUT_ARGUMENT {
      public static final String Text = "text";
    }
  }

  public static class HUMAN {
    public static final String TYPE_NAME = "Human";

    public static final String Id = "id";

    public static final String Name = "name";

    public static final String TotalCredits = "totalCredits";
  }

  public static class DROID {
    public static final String TYPE_NAME = "Droid";

    public static final String Id = "id";

    public static final String Name = "name";

    public static final String PrimaryFunction = "primaryFunction";
  }

  public static class SEARCHRESULTPAGE {
    public static final String TYPE_NAME = "SearchResultPage";

    public static final String Items = "items";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/unionTypesWithoutInterfaceCanDeserialize/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.unionTypesWithoutInterfaceCanDeserialize.expected

import kotlin.String

@Generated
public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  @Generated
  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Search: String = "search"

    @Generated
    public object SEARCH_INPUT_ARGUMENT {
      public const val Text: String = "text"
    }
  }

  @Generated
  public object HUMAN {
    public const val TYPE_NAME: String = "Human"

    public const val Id: String = "id"

    public const val Name: String = "name"

    public const val TotalCredits: String = "totalCredits"
  }

  @Generated
  public object DROID {
    public const val TYPE_NAME: String = "Droid"

    public const val Id: String = "id"

    public const val Name: String = "name"

    public const val PrimaryFunction: String = "primaryFunction"
  }

  @Generated
  public object SEARCHRESULTPAGE {
    public const val TYPE_NAME: String = "SearchResultPage"

    public const val Items: String = "items"
  }

  @Generated
  public object SEARCHRESULT {
    public const val TYPE_NAME: String = "SearchResult"
  }
}
